engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableFaceCulling.java

```diff
@@ -19,7 +19,10 @@
 import org.terasology.rendering.dag.StateChange;
 
 /**
- * Instances of this class enable OpenGL's Face Culling, i.e. to render only the inside faces of the skysphere.
+ * Enables OpenGL's Face Culling.
+ *
+ * This can be used to discard the fragments that face away from the camera, and can be used to improve
+ * the performance in certain situations like rendering only the inside faces of the skysphere.
```

Comment:
Using this as it is, with a small change:
`Face Culling is used to discard triangles facing -a particular direction-, reducing..`
to make things more generic.